dashboard/app/controllers/sprite_management_controller.rb

```diff
@@ -0,0 +1,4 @@
+class SpriteManagementController < ApplicationController
```

Comment:
Should we require levelbuilder permissions?

Reply:
Good call! Updated!